flow-decomposition/src/main/java/com/powsybl/flow_decomposition/LossesCompensator.java

```diff
@@ -47,7 +47,7 @@ void run(Network network) {
         compensateLossesOnBranches(network);
     }
 
-    private void addZeroMWLossesLoadsOnBuses(Network network) {
+    static void addZeroMWLossesLoadsOnBuses(Network network) {
```

Comment:
breaks encapsulation and make this class usage strange as it is both a runner object (run member method) and a static utility class (I do not know is run method is still used by the way)

Reply:
fixed